dask/dataframe/core.py

```diff
@@ -5086,6 +5091,40 @@ def applymap(self, func, meta="__no_default__"):
     def round(self, decimals=0):
         return elemwise(M.round, self, decimals)
 
+    @derived_from(pd.DataFrame)
+    def nunique(self, split_every=False, dropna=True, axis=0):
```

Comment:
Maybe a note that `split_every` is not relevant for `axis=1` would be helpful

Reply:
added!